charts/sqnc-node/values.yaml

```diff
@@ -1,173 +1,388 @@
+    ##
+    createP2pService: true
+    ## @param node.perNodeServices.p2pServiceType Must be type ClusterIP, NodePort or LoadBalancer. If using type NodePort or LoadBalancer, then you must set NodeSelector accordingly
+    ##
+    p2pServiceType: ClusterIP
+    setPublicAddressToExternal:
+      ## @param node.perNodeServices.setPublicAddressToExternal.enabled If true sets the `--public-addr` flag to be the NodePort P2P services external address
+      ##
+      enabled: true
+      ## @param node.perNodeServices.setPublicAddressToExternal.ipRetrievalServiceUrl The external service to return the NodePort IP
+      ##
+      ipRetrievalServiceUrl: https://ifconfig.io/ip
+  ## @param node.keys The list of keys to inject on the node before startup (object{ type, scheme, seed })
+  ##
+  ## Example
+  ## keys:
+  ##  - type: "gran"
+  ##    scheme: "ed25519"
+  ##    seed: "//Alice//gran"
+  ##  - type: "babe"
+  ##    scheme: "sr25519"
+  ##    seed: "//Alice//babe"
+  ##
+  keys: {}
+  ## @param node.persistGeneratedNodeKey [default: false] Persist the auto-generated node key inside the data volume (at /data/node-key)
+  ##
+  persistGeneratedNodeKey: false
+  ## @param node.customNodeKey [default: nil] Use a custom node-key, if node.persistGeneratedNodeKey is true then this will not be used. (Must be 64 byte hex key)
+  ##
+  ## customNodeKey: ""
+  ##
+  ## @param node.enableStartupProbe [default: true] If true, enable the startup probe check
+  ##
+  enableStartupProbe: true
+  ## @param node.enableReadinessProbe [default: true] If true, enable the readiness probe check
+  ##
+  enableReadinessProbe: true
+  ## @param node.dataVolumeSize [default: 100Gi] The size of the chain data PersistentVolume
+  ##
+  dataVolumeSize: 10Gi
+  ## @param node.replicas [default: 1] Number of replicas in the node's StatefulSet
+  ##
+  replicas: 1
+  ## @param node.role [default: full] Set the role of the node: full, authority/validator, collator or light
+  ##
+  role: full
+  ##
+  ## @param node.chainDataSnapshotUrl [default: ""] Download and load chain data from a snapshot archive http URL
+  ## @param node.chainDataSnapshotFormat [default: tar] The snapshot archive format (tar or 7z)
+  ## @param node.chainPath [default: nil] Path at which the chain database files are located (/data/chains/${CHAIN_PATH})
+  ## @param node.chainDataKubernetesVolumeSnapshot [default: ""] Initialize the chain data volume from a Kubernetes VolumeSnapshot
+  ## @param node.chainDataGcsBucketUrl [default: ""] Sync chain data files from a GCS bucket (eg. gs://bucket-name/folder-name)
+  ##
+  ## chainDataSnapshotUrl: "https://dot-rocksdb.polkashots.io/snapshot"
+  ## chainDataSnapshotFormat: 7z
+  ## chainPath: ""
+  ## chainDataKubernetesVolumeSnapshot: ""
+  ## chainDataGcsBucketUrl: ""
+  ##
+  ## @param node.customChainspecUrl [default: nil] Download and use a custom chainspec file from a URL
+  ## customChainspecUrl:
+  ##
+  collator:
```

Comment:
not this ticket but we need to clean up this chart. We don't do anything related to collators or parachains